src/index.tsx

```diff
@@ -32,7 +32,7 @@ const root = (
 
 ReactDOM.render(root, document.getElementById('root') as HTMLElement);
 
-const isDesktop = typeof carlo !== undefined;
+const isDesktop = typeof carlo !== 'undefined';
```

Comment:
- [`typeof` operator always returns a string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof)
- already in `master`
- 1.0 shipped with a broken service worker because of this

Reply:
TIL. JS being unexpected. Wonder why TS didn't flag this.